engines/suppliers/app/models/suppliers/orders_frequency.rb

```diff
@@ -0,0 +1,32 @@
+require 'ice_cube'
+
+module Suppliers
+  class OrdersFrequency < ActiveRecord::Base
+    self.table_name = :orders_frequencies
+
+    FREQUENCY_TYPES = { confirmation: 0, delivery: 1 }.freeze
```

Comment:
we have 2 kind of recurrencies related to orders:

1. when will be the next limit date/time to confirm an order? (type `:confirmation`)
2. when will be the next delivery date? (type `:delivery`)

It's not a stage or status of the order. We use this just to set dates while creating a new order.